<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>381</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="listNote"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>160</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListView" name="listCategory"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuNouveau">
     <property name="title">
      <string>Nouveau</string>
     </property>
     <addaction name="actionNewBlocNote"/>
     <addaction name="actionNewNote"/>
    </widget>
    <widget class="QMenu" name="menuSupprimer">
     <property name="title">
      <string>Supprimer</string>
     </property>
     <addaction name="actionDelBlocNote"/>
     <addaction name="actionDelNote"/>
    </widget>
    <addaction name="menuNouveau"/>
    <addaction name="menuSupprimer"/>
    <addaction name="separator"/>
    <addaction name="actionMini"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acToHTML">
   <property name="text">
    <string>La note sélectionnée ves HTML</string>
   </property>
  </action>
  <action name="acAllToHTML">
   <property name="text">
    <string>Toutes les notes vers HTML</string>
   </property>
  </action>
  <action name="actionNewBlocNote">
   <property name="text">
    <string>Bloc-Note</string>
   </property>
  </action>
  <action name="actionNewNote">
   <property name="text">
    <string>Note</string>
   </property>
  </action>
  <action name="actionDelBlocNote">
   <property name="text">
    <string>Bloc-Note</string>
   </property>
  </action>
  <action name="actionDelNote">
   <property name="text">
    <string>Note</string>
   </property>
  </action>
  <action name="actionMini">
   <property name="text">
    <string>Mini</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
